### PR TITLE
Created a README and updated the text in the body

### PR DESCRIPTION
**Pull request by Obinnabii**
This is the initial tutorial and README.

### CODE REVIEW

**Comment by Obinnabii** [comment]
I thought it would start a PR every time I pushed :(
I used today to brainstorm ideas for the actual website and try out some cool JS features.

**Comment by Obinnabii** [comment]
Is there a way to make it do that with pushes, or do I have to manually start PRs every time I push?

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
+###The Projects
 
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
+1. NaN		0%
+2. NaN		0%
+3. NaN		0%
+4. NaN		0%
```
Comment (Obinnabii) [inline README.md:11]: I used NaN as a placeholder because I didn't know what the projects would be called.

Comment (billyeh) [reply README.md:11]: Great! Often we use TODO comments to let us and others know we will or should address something in the future.

**Comment by billyeh** [comment]
Sounds like good progress for today! Unfortunately I'm not aware of a way to automatically open pull requests like that, other than maybe another Github CLI tool? https://github.com/github/hub

Basically just need to hook up Git committing with hitting the Github pull request API, which that project might help with.

In general I think people work on their branch, pushing commits until it's "ready" and then open a pull request.

**Review by billyeh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Obinnabii) [reply README.md:11]: Sure thing, I'll add those!

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ##Obi Abii's __portfolio__
 
-This repo contains Obi's *STEP projects*. As of now, I have implemented 0 of the projects;
+This repo contains Obi's *STEP projects*. As of now, I have implemented 0 of the projects.
```
Comment (Obinnabii) [inline README.md:5]: Thanks!

**Pull request merged**